tests/robustness/model/history.go

```diff
@@ -493,54 +445,57 @@ func defragmentResponse(revision int64) MaybeEtcdResponse {
 }
 
 type History struct {
-	successful []porcupine.Operation
-	// failed requests are kept separate as we don't know return time of failed operations.
-	// Based on https://github.com/anishathalye/porcupine/issues/10
-	failed []porcupine.Operation
+	operations []porcupine.Operation
 }
 
 func (h History) Merge(h2 History) History {
 	result := History{
-		successful: make([]porcupine.Operation, 0, len(h.successful)+len(h2.successful)),
-		failed:     make([]porcupine.Operation, 0, len(h.failed)+len(h2.failed)),
+		operations: make([]porcupine.Operation, 0, len(h.operations)+len(h2.operations)),
 	}
-	result.successful = append(result.successful, h.successful...)
-	result.successful = append(result.successful, h2.successful...)
-	result.failed = append(result.failed, h.failed...)
-	result.failed = append(result.failed, h2.failed...)
+	result.operations = append(result.operations, h.operations...)
+	result.operations = append(result.operations, h2.operations...)
```

Comment:
check the order of and result[last] and h2[0] first.

Reply:
Oh, nice catch, however this function is no longer used so let's just remove it.